.gitignore

```diff
@@ -151,4 +184,6 @@ build
 
 outputData/
 
-logs/
\ No newline at end of file
+logs/
+
+input/mobilitySimulatorMid/
```

Comment:
Well this one is probably only going to be useful to you...

There is a trick with `.gitignore`s though: Place a `.gitignore` inside `input/mobilitySimulatorMid/` with the content `*`. That way, the directory is ignored

Reply:
did not know about that - thanks